README.md

```diff
@@ -7,6 +7,8 @@ If you're running Linux
 sudo apt-get install libunwind8
 ```
 
+The autorest version we are using only works up to node v14.
```

Comment:
Mostly cosmetic suggestion to use capitalized tool names and include a link to the AutoRest project page on GitHub (can be useful for people who never used it before):

```suggestion
The [AutoRest](https://github.com/Azure/autorest) version we are using only works up to Node.js v14.
```